src/lib/get-token.js

```diff
@@ -18,6 +19,43 @@ async function getToken(store, client, auth, state) {
     scope: getScopeForResponse(client, auth.scope)
   };
 
+  const scopes = auth.scope.split(' ');
+
+  if (scopes.includes('openid')) {
+    const provider = process.env.PROVIDER_URL || 'http://localhost:3020/';
+    const secret = process.env.SECRET || 'uber5';
+    const now = new Date();
+    const { identifiers } = await store.getUserById(auth.userId);
+    // eslint-disable-next-line one-var
+    let email, phone;
```

Comment:
This is hard-coding that we support/know identifiers called `email` and `phone`. Why?

We should (and can) implement this in such a way that whatever identifiers there are, we'll provide them as part of the id token payload.

Then, if we ever get an identifier `face:12345`, we would automatically support `face` as an identifier key, and provide its id `12345` as part of the userinfo.

Reply:
I managed to make it dynamic but still some improvements..